functions/index.js

```diff
@@ -47,10 +47,14 @@ function sendNotification(tokenArray) {
 
 function sendPayload(tokenArray) {
   
-  var payload = {
-    notification: {
+  const payload = {
+    data: {
       title: "Meggin needs milk",
-      body: "Can you get Meggin milk?"
+      body: "Can you get Meggin milk?",
+      actions: [
```

Comment:
This isnt' valid in the FCM API (i.e. you van only send key value pairs where the value is a string.).

What you can do is:

```javascript
const payload = {
  data: {
    title: '',
    notificationOptions: JSON.stringify({
      body: '',
      actions:[{...}, {...}]
    })
  }
};
```

(See relevant section in sw to handle over side of this change)